test/homework_test/01_variables_test/01_variables_tests.cpp

```diff
@@ -12,3 +12,16 @@ TEST_CASE("Verify sum_numbers function")
 	REQUIRE(add_numbers(10, 10) == 20);
 }
 
+TEST_CASE("verify the calling of get_sales_tax_amount function")
+{
+	REQUIRE(get_sales_tax_amount(10)==.675);
+	REQUIRE(get_sales_tax_amount(20)==1.35);
+}
+
+TEST_CASE("verify the calling of get_tip_amount function")
+{
+	REQUIRE(get_tip_amount(20,15));
+	REQUIRE(get_tip_amount(20,20));
```

Comment:
Should be
REQUIRE(get_tip_amount(20,15) == 3);
REQUIRE(get_tip_amount(20,20) == 4);